scripts/sysman/memory.yml

```diff
@@ -137,6 +137,10 @@ class: $sMemory
 name: $s_mem_bandwidth_t
 details:
     - "Percent bandwidth is calculated by taking two snapshots (s1, s2) and using the equation: %bw = 10^6 * ((s2.readCounter - s1.readCounter) + (s2.writeCounter - s1.writeCounter)) / (s2.maxBandwidth * (s2.timestamp - s1.timestamp))"
+    - "Counter may be 32 bit or 64 bit depends on the platform and rolls over when the count reaches 2^32 or 2^64. Rollover needs to be handled by comparing the current read against the previous read"
```

Comment:
this seems to be very implementation specific. Why do we need it in the spec?

Reply:
Its not implementation specific, its general about counters and and its beheviours